### DISCUSSION TITLE
Project Proposal: CodeCamp

### DISCUSSION THREAD

**Discussion by mehakagg1313** [Tech-Phantoms/Maintainers-Program#16] [Application]
About me-
I am Mehak Aggarwal.
I am an Engineering Undergraduate studying in the branch computer science in artificial intelligence from the university, Indira Gandhi Delhi Technical University for Women.
I love networking and connecting with people from different backgrounds and hence am an active member of a lot of technical as well as non-technical societies .

some **projects** I have worked on
.Hindi Fake News Dataset and Analysis , Link - [https://github.com/mehakagg1313/Internship_Project]
.Halloween E-Commerce Website , Link - [https://github.com/mehakagg1313/Halloween_Website]
website -[https://scary-halloween.web.app/]
. IMAGE CAPTION GENERATOR, Link - [https://github.com/mehakagg1313/IT_PROJECT]
more projects you can check on my github
**social links**
email id- [mehakagg1313@gmail.com]
instagram- [https://www.instagram.com/_.mehak__12/]
linkedin - [https://www.linkedin.com/in/mehak-aggarwal-97315a1b4/]
**Project Proposal**
https://docs.google.com/document/d/1nuXVhsp7eE8Cd9L2h_13GLsCCAtPmP1ZZQzDWDdToj8/edit?usp=sharing

**Comment by imabp** [comment]
Hey @mehakagg1313 , we would like to invite you to our discord.
Please reply back here, with discord id once joined.

[Join Discord](https://discord.apihacks.co)

**Reply by mehakagg1313** [reply]
discord id- mehak#2545
